### ISSUE TITLE
[Question] Fine-tuned model ignore some of the captions 

### ISSUE THREAD

**Issue by AbdulrahmanSoliman1** [haotian-liu/LLaVA#1637]
### Question
![image](https://github.com/user-attachments/assets/d10a4267-cf27-4156-8651-0def332f9f45)
train/loss 0.0001 , epochs 1
the model always focuses on one part of the captions and ignores the rest. I am using the default fine-tuning
the dataset is 20k images-captions

Ground Truth: Medical image shows broken leg and lighting could impact visibility.
LLAVA: Medical image shows broken leg.

is there something to do that makes the model focus on the whole label

def __getitem__(self, idx):
    sample = self.data[idx]
    image_path = os.path.join(self.image_dir, sample["image_file"]) 
    image = Image.open(image_path).convert("RGB")
    return {
        "image": image,
        "qa": [
            {
                "question": "Describe the following image in detail",
                "answer": sample["description"],
            }
        ]
    }
